Tools/Create-Azure-Sentinel-Solution/input/Solution_DigitalShadowsSearchlight.json

```diff
@@ -0,0 +1,15 @@
+ {
+    "Name": "DigitalShadowsSearchlight",
+    "Author": "{Angus Gordon - angus.gordon@digitalshadows.com}",
+    "Logo": "<img src=\"{LogoLink}\" width=\"75px\" height=\"75px\">",
```

Comment:
Please add raw logo link in place of {LogoLink} , thanks.

Reply:
image in base64 format, right?

Reply:
see example: https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/examples/Solution_ExampleInput.json

You can just add the logo to the repository and link to it from there. I can send you a SVG logo if required

Reply:
@shikhin-metron, Replace {LogoLink} with " https://raw.githubusercontent.com/parag-metron/Azure-Sentinel/Digital-Shadows-connector/Logos/DigitalShadowsLogo.svg ", thanks.

Reply:
Do we want that link to be pointing at " https://raw.githubusercontent.com/Azure/Azure-Sentinel/Digital-Shadows-connector/Logos/DigitalShadowsLogo.svg " so that once this PR is merged it is pointing to the correct location?

Reply:
> @shikhin-metron, Replace {LogoLink} with " https://raw.githubusercontent.com/parag-metron/Azure-Sentinel/Digital-Shadows-connector/Logos/DigitalShadowsLogo.svg ", thanks.

@v-sabiraj so I am guessing I will have to create such folder and keep a logo in it in next commit? We have a logo folder in `Solutions/Data Connectors` folder as well btw.